lib/bridgetown-inline-svg/tag.rb

```diff
@@ -0,0 +1,62 @@
+module BridgetownInlineSvg
+  class Tag < Bridgetown::Builder
+    attr_reader :attributes
+    attr_reader :context
+    attr_reader :markup
+    attr_reader :svg_path
+
+    def build
+      liquid_tag "svg", :render
+    end
+
+    def render(markup, builder)
+      @context = builder.context
+      @markup = markup
+
+      interpolate_variables_in_markup!
+      set_svg_path_and_attributes!
+
+      return unless svg_path
```

Comment:
I was wondering if we should throw a loud error here if the file is missing, what do you think?

Reply:
I guess I'd like to see what that would look like before deciding. Maybe you could open a PR to [the starter](https://github.com/andrewmcodes/bridgetown-inline-svg-starter) and change the gem source to this branch and screen shot what that would look like?